website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -13,9 +14,18 @@ import {
   getTasks,
   moveTask,
 } from '../../../libs/taskManager';
+import apiMessages from '../../../libs/apiMessages';
 
 let requiredGroupFields = '_id leader tasksOrder name';
 let types = Tasks.tasksTypes.map(type => `${type}s`);
+
+function canNotEditTasks (group, user, assignedUserId) {
```

Comment:
If we start abstracting permissions for routes, I'd rather have a full separate PR on creating "policy" services in a unified way.